### PR TITLE
Create oc.yml

### PR DESCRIPTION
**Pull request by Quent-in**
Translated in General Occitan

### CODE REVIEW

**Comment by yookoala** [comment]
This file suffers multiple issues. You'd need to help fix them before the translation can ever be used.

1. Indentation: YAML is structured with space indentation. Each level will differ themselves from the upper one with 2-spaces. The problems I found:
    1. Some of your lines are indented with `tab` instead of `space`.
    2. Some of the entries are not indented the same as `en.yml`.

2. Wrong format: Some of the keys are found duplicated in the same level. This indicates that you've edited the file wrongly, indented wrongly, or re-ordered the lines by mistake.

3. Keys outdated: Some of the contents you translated belongs to older version of `en.yml`. Please start with the latest version of it, then try to merge your translation back in.

File: config/locales/oc.yml [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    error: O planhèm, i a agut una error al moment de cercar lo compte
+    follow: Sègre
+    prompt_html: 'Avètz (<strong>%{self}</strong>) demandat de sègre :'
+    title: Sègre %{acct}
+  datetime:
+    distance_in_words:
+      about_x_hours: "%{count}oras"
+      about_x_months: "%{count}meses"
+      about_x_years:
+		one: un an
```
Comment (yookoala) [inline config/locales/oc.yml]: Example of lines indented with tabs instead of space

File: config/locales/oc.yml [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        title: Description étendue du site
+        desc_html: "Affichée sur la page d'informations complémentaires du site<br>Vous pouvez utiliser des balises HTML"
+      registrations:
+        open:
+          title: Inscripcions
+          enabled: Activadas
+          disabled: Desactivadas
+        closed_message:
+          title: Message de fermeture des inscriptions
+desc_html: "Affiché sur la page d'accueil lorsque les inscriptions sont fermées<br>Vous pouvez utiliser des balises HTML"
```
Comment (yookoala) [inline config/locales/oc.yml]: Example of incorrect indentation.

**Comment by yookoala** [comment]
@Quent-in: Should have got most issues fixed in #2077.

(There are missing translations that you'd need to fill in later)

**Comment by Quent-in** [comment]
Hello and thanks for the review!
So I have to look at en.yml and find the differences?
What about "app/assets/javascripts/components/locales/oc.jsx", I thought its format would be ok.
Have all my files syntax errors?
How do I send again my files with the same Pull Request?
Many thanks!

**Comment by yookoala** [comment]
#2077 Got merged. That includes all your work (include oc.yml and oc.jsx). If you're doing local git command, please use: 

```
git fetch origin
git reset --hard origin/master
```

to checkout the latest source code. See if I mishandled anything.

**Comment by Quent-in** [comment]
I'll try your advice for the YAML editor, I'll complete the translation.
Do I have to compare the differences with the "en" files, or are the news strings automatic put in other languages?
Many thanks!

**Comment by yookoala** [comment]
English YAML is always updated. English JSX might be outdated, but would catch up soon.

So my advice would be to compare your file with the English ones.

**Pull request closed without merging**